hera_mc/cm_handling.py

```diff
@@ -363,23 +382,30 @@ def get_part_dossier(self, hpn, rev, at_date, exact_match=False, full_version=Tr
         hpn:  the input hera part number [string or list-of-strings] (whole or first part thereof)
         rev:  specific revision(s) or category(ies) ('LAST', 'ACTIVE', 'ALL', 'FULL', specific)
               if list, must match length of hpn
-        at_date:  reference date of dossier only used for ACTIVE
+        at_date:  reference date of dossier (and stop_date for displaying notes)
```

Comment:
I thought we discussed this and decided not to conflate these two concepts into one keyword.

Reply:
I spent some time with this and decided that keeping the 'at_date' and changing 'early' to 'start_date' was more clear in actual use.  Otherwise, if you want to filter on notes you have to add another identical date to the call (so you'd have two of the same), which seems silly since the context is relatively clear.  The other approach would be to rename 'at_date' to 'start_date', which would make it very confusing in every context, including this one.

Reply:
ok. I'll let your actual users weigh in via issues if they find it confusing.